### ISSUE TITLE
Can you provide more information on the details of the training?

### ISSUE THREAD

**Issue by Ftiasch1** [Hao840/OFAKD#6]
Thanks to the authors for their excellent work. However, there are some problems when trying to reproduce the distillation methods documented in the paper, and it is difficult to reproduce them to the documented accuracy. I wonder if the authors can provide information on the hyperparameters of each distillation method during training.

**Comment by Hao840** [comment]
The default hyperparameters of each method are those listed in the `train.py`. Sometimes the cross-architecture distillation is unstable, and you can try to decrease the weight of distillation loss by adjusting `--kd-loss-weight` or use smaller `--ofa-eps` value.

**Comment by liguopeng0923** [comment]
I have the same problem. I guess that is caused by different numbers of gpus. And the hyperparameters need to be adjusted carefully.

**Comment by Hao840** [comment]
All our experiments are conducted with 8 GPUs, which indicates that the real batch size is `8*args.batch_size`. A smaller batch size setting may lead to a worse estimation of gradients and thus result in a worse performance.

**Comment by Ftiasch1** [comment]
> The default hyperparameters of each method are those listed in the `train.py`. Sometimes the cross-architecture distillation is unstable, and you can try to decrease the weight of distillation loss by adjusting `--kd-loss-weight` or use smaller `--ofa-eps` value.

Thank you for your response.

**Comment by Ftiasch1** [comment]
> I have the same problem. I guess that is caused by different numbers of gpus. And the hyperparameters need to be adjusted carefully.

I followed the author's settings on GPU number and Batch Size. I was able to get comparable results on the ImageNet-1K dataset, but had problems with the Cifar100 reproduction.

**Comment by Ftiasch1** [comment]
> I have the same problem. I guess that is caused by different numbers of gpus. And the hyperparameters need to be adjusted carefully.

I followed the author's settings on GPU number and Batch Size. I was able to get similar results on the ImageNet-1K dataset, but had problems with the Cifar100 reproduction.

**Comment by liguopeng0923** [comment]
> > I have the same problem. I guess that is caused by different numbers of gpus. And the hyperparameters need to be adjusted carefully.
> 
> I followed the author's settings on GPU number and Batch Size. I was able to get similar results on the ImageNet-1K dataset, but had problems with the Cifar100 reproduction.

Fine, it is difficult for me to train it from scratch. I need to find a better solution.
